### PR TITLE
[fix] add test folder to includes

### LINKED ISSUE #5833: Typescript warns that a .svelte file import into a test file doesn't exist.
### Describe the bug

I'm getting typescript warnings that a `.svelte` file that I'm importing in a test file doesn't exist. But `vitest` runs fine. It seems to work if I add an includes in the project `tsconfig.json` to 

```
"include": [
    "./.svelte-kit/ambient.d.ts",
    "src/**/*.js",
    "src/**/*.ts",
    "src/**/*.svelte",
    "src/**/*.js",
    "src/**/*.ts",
    "src/**/*.svelte",
    "test/**/*.ts",
    "test/**/*.js",
    "test/**/*.svelte"
  ],
```

and relaunch VS Code. But I'm wondering why test directory wasn't included in .`svelte-kit/tsconfig.json` in the first place.

Screenshot 2022-08-05 at 14 59 39

### Reproduction

https://github.com/manuganji/sjsf/tree/17b0742b3d25347a6c4be2186a7bd8f568ea32ae

### Logs

_No response_

### System Info

```shell
System:
    OS: macOS 12.1
    CPU: (8) x64 Apple M1
    Memory: 30.89 MB / 8.00 GB
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 16.14.2 - ~/.nvm/versions/node/v16.14.2/bin/node
    Yarn: 1.22.17 - /opt/homebrew/bin/yarn
    npm: 8.10.0 - ~/.nvm/versions/node/v16.14.2/bin/npm
  Browsers:
    Chrome: 104.0.5112.79
    Firefox: 100.0.2
    Safari: 15.2
  npmPackages:
    @sveltejs/adapter-auto: next => 1.0.0-next.63 
    @sveltejs/kit: next => 1.0.0-next.403 
    svelte: ^3.44.0 => 3.49.0 
    vite: ^3.0.0 => 3.0.3
```

### Severity

annoyance

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by dummdidumm**
Also add some notes to tsconfig/jsonfig
Fixes #5833

I briefly thought about making this configurable through `kit.files`, but didn't do so because only the tsconfig includes generation would benefit from it - you'd still need to update the package.json scripts yourself.

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [x] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [x] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [x] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

File: .changeset/hot-lions-cough.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'create-svelte': patch
+---
+
+Add notes about includes/excludes and path aliases
```
Comment (benmccann) [inline .changeset/hot-lions-cough.md:5]: personally i'd remove this changeset as we don't typically add them for documentation changes

Comment (Conduitry) [reply .changeset/hot-lions-cough.md:5]: It's not documentation for the site, though, is it? I think this adds a comment to what people actually get when they run `create-svelte`, which would warrant a changeset.

Comment (benmccann) [reply .changeset/hot-lions-cough.md:5]: ah, yeah. fair enough

File: packages/create-svelte/shared/+typescript/tsconfig.json [hunk 2]
```diff
@@ -10,4 +10,8 @@
 		"sourceMap": true,
 		"strict": true
 	}
+	// Path aliases are handled by https://kit.svelte.dev/docs/configuration#alias and https://kit.svelte.dev/docs/configuration#files
```
Comment (benmccann) [inline packages/create-svelte/shared/+typescript/tsconfig.json:13]: I don't think we need the reference to https://kit.svelte.dev/docs/configuration#files. The only thing there is `$lib` and that's already mentioned in https://kit.svelte.dev/docs/configuration#alias

```suggestion
	// Path aliases are handled by https://kit.svelte.dev/docs/configuration#alias
```

Comment (dummdidumm) [reply packages/create-svelte/shared/+typescript/tsconfig.json:13]: Good point

File: packages/kit/src/core/sync/write_tsconfig.js [hunk 3]
```diff
@@ -53,6 +53,12 @@ export function write_tsconfig(config, cwd = process.cwd()) {
 		include.push(config_relative(`${relative}/**/*.ts`));
 		include.push(config_relative(`${relative}/**/*.svelte`));
 	}
+	// Test folder is a special case - we advocate putting tests in a top-level test folder
+	// and it's not configurable (should we make it?)
+	const test_folder = project_relative('test');
```
Comment (dominikg) [inline packages/kit/src/core/sync/write_tsconfig.js:58]:
```suggestion
	const test_folder = project_relative('tests');
```

**Review by Rich-Harris — APPROVED** [review]

**Pull request merged**